blazor/chart/data-markers.md

```diff
@@ -93,6 +93,63 @@ Markers can be assigned with different shapes such as [Rectangle](https://help.s
 
 ![Blazor Line Chart with Diamond Marker](images/marker/blazor-line-chart-diamond-marker.png)
 
+## Display different markers for each chart series
+
+When more number of series availabale and the marker is enabled, then each chart series will display different marker shapes automatically.
```

Comment:
Change the header and content

## Auto Marker Shape

By default, the chart marker shape is set to Auto. If chart marker is enabled, each series will be rendered with a different marker shape.